mlflow/store/tracking/sqlalchemy_store.py

```diff
@@ -142,7 +141,6 @@ def __init__(self, db_uri, default_artifact_root):
         # DB migrations
         if not mlflow.store.db.utils._all_tables_exist(self.engine):
             mlflow.store.db.utils._initialize_tables(self.engine)
-        Base.metadata.bind = self.engine
```

Comment:
/ai is this valid sqlalchemy code?

Reply:
/ai but why after removing this line code, all unit tests related to this still passed ?

Reply:
@WeichenXu123 It's because it's useless.